src/lib.rs

```diff
@@ -16,10 +16,11 @@
 #![no_std]
 
 use core::alloc::{GlobalAlloc, Layout};
+use core::cmp;
 use core::ffi::c_void;
 use core::ptr;
 
-#[cfg(target_family = "unix")]
+#[cfg(any(target_family = "unix", feature = "libc"))]
```

Comment:
Now that I think about it... maybe this should just be

```rust
#[cfg(not(target_family = "windows"))]
```

and avoid introducing the new feature entirely?

Reply:
I prefer your suggestion.